modules/sdk-coin-tia/src/lib/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { InvalidTransactionError } from '@bitgo/sdk-core';
+import { Coin } from '@cosmjs/stargate';
+import BigNumber from 'bignumber.js';
+
+import { CosmosUtils } from '@bitgo/abstract-cosmos';
+import * as constants from './constants';
+
+export class TiaUtils extends CosmosUtils {
+  /** @inheritdoc */
+  isValidAddress(address: string): boolean {
+    return constants.accountAddressRegex.test(address);
+  }
+
+  /** @inheritdoc */
+  isValidValidatorAddress(address: string): boolean {
+    return constants.validatorAddressRegex.test(address);
+  }
+
+  /** @inheritdoc */
+  validateAmount(amount: Coin): void {
```

Comment:
similar to osmo: https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-coin-osmo/src/lib/utils.ts#L20